app/views/organizations/adopter_fosterer/fostered_pets/index.html.erb

```diff
@@ -13,7 +13,13 @@
                                   class: 'rounded-circle', 
                                   style: "width: 100%; height: 100%; object-fit: cover;") %>
                   </div>
-                  <h5 class="card-title mb-0"><%= fostered_pet.pet.name %></h5>
+                  <h5 class="card-title mb-0 flex-grow-1" style="margin-right: 15px;"><%= fostered_pet.pet.name %></h5>
```

Comment:
If you just use flex on this you should be able to use flex classes like `justify-content-around` to space these main elements evenly in the parent div https://getbootstrap.com/docs/4.0/utilities/flex/#justify-content

You should not need to use inline style here like `style="margin-right: 15px;"` as Bootstrap has classes for this.